openthread/Kconfig

```diff
@@ -81,16 +75,12 @@ config OPENTHREAD_NORDIC_LIBRARY_FTD
 	bool "Nordic Optimized FTD configuration"
 	select OPENTHREAD_COAP
 	select OPENTHREAD_DNS_CLIENT
-	select OPENTHREAD_DHCP6_CLIENT
 	select OPENTHREAD_ECDSA
 	select OPENTHREAD_EXTERNAL_HEAP
 	select OPENTHREAD_IP6_FRAGM
-	select OPENTHREAD_JOINER
 	select OPENTHREAD_NETDIAG_CLIENT
 	select OPENTHREAD_SLAAC
-	select OPENTHREAD_SNTP_CLIENT
 	select OPENTHREAD_SRP_CLIENT
-	select OPENTHREAD_DUA if !OPENTHREAD_THREAD_VERSION_1_1
```

Comment:
Please make sure Thread 1.2 Certification 5.9.x tests are optional.

Reply:
We'll discuss that on today's meeting with GRL

Reply:
Discussion on DUA has been brought up to Thread Group, but will be continued separately. Reverted removing `OPENTHREAD_DUA` from this PR.